recipes/capstone/all/conanfile.py

```diff
@@ -35,18 +36,18 @@ class CapstoneConan(ConanFile):
         "fPIC": True,
         "use_default_alloc": True,
     }
+    implements = ["auto_shared_fpic"]
 
     _archs = ["arm", "m68k", "mips", "ppc", "sparc", "sysz", "xcore", "x86", "tms320c64x", "m680x", "evm"]
     options.update({a: [True, False] for a in _archs})
     default_options.update({a: True for a in _archs})
 
+    def _is_universal2(self):
```

Comment:
Where is this function being called? I don't see if referenced in the diff

Reply:
>I do still think for adding those lines into & placing
tc.cache_variables["CAPSTONE_BUILD_MACOS_THIN"] = False  # Enable universal2 builds on macOS later on

    def package_id(self):
        if self._is_universal2(info=True):
            self.info.settings.arch = "universal:armv8/x86_64"
It is not being called. I can safely remove it.
We will decide on it later. I just never experienced working with universal2.

Reply:
@AbrilRBS 
https://github.com/conan-io/conan-center-index/blob/037efda2eedecaf2b550b9dfbc9d522cb8f6e56a/recipes/android-ndk/all/conanfile.py#L52-L56